src/mmrelay/meshtastic_utils.py

```
@@ -570,21 +623,72 @@ def connect_meshtastic(
+                                    "Using forked meshtastic library with auto_reconnect support"
+                                )
+                            else:
+                                logger.debug("Using official meshtastic library")
+
+                            try:
+                                meshtastic_iface = (
+                                    meshtastic.ble_interface.BLEInterface(**ble_kwargs)
+                                )
+                            except Exception as e:
+                                # BLEInterface constructor failed - this is a critical error
+                                logger.exception(f"BLE interface creation failed: {e}")
```

Ruff (0.14.10)

Redundant exception object included in `logging.exception` call

(TRY401)